src/widgets/DictionaryWidget/DictionaryContainer.jsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { AiFillSound } from 'react-icons/ai';
```

Comment:
error in this line

Reply:
Okay let me check why is it coming?

Reply:
you need to upload package.json as it does not contain this package react-icon

Reply:
Okay, I'll do that